### PR TITLE
Cleanup spec file and remove ssl directory from git root

### PR DESCRIPTION
**Pull request by felixsch**

### CODE REVIEW

**Comment by Sergeykot** [comment]
screenshot_20180831_173649
FYI when I install RMT now on version 1.0.0 which has no generated SSL certs I get this output

**Comment by Sergeykot** [comment]
FYI (maybe will help to debug): before this PR the output was this one:
screenshot_20180831_173728

File: package/rmt-server.spec [hunk 1]
```diff
@@ -234,9 +233,9 @@ fi
 
 # Run only on upgrade
 if [ $1 -eq 2 ]; then
-  if [ -d %{app_dir}/ssl ]; then
-    mv %{app_dir}/ssl/* %{conf_dir}/ssl
-    echo "RMT ssl configuration has been moved to a new place. New place is: %{conf_dir}/ssl"
+  if [ -d %{app_dir}ssl ]; then
+    mv %{app_dir}ssl/* %{conf_dir}ssl
+    echo "RMT ssl configuration has been moved to a new place: %{conf_dir}/ssl"
```
[the remark this answers is not in the document]
Comment (felixsch) [reply package/rmt-server.spec]: :+1:

**Comment by felixsch** [comment]
@Sergeykot @ikapelyukhin The problem is that all installations moved the ssl directory from the git repository which has the .keep file. That's why it's moving even if there are no ssl certificates generated yet. I can add to the check for `rmt-ca.crt` to make sure that the certificates exist and leave behind a stale directory in case it's not. What do you think?

**Comment by ikapelyukhin** [comment]
@felixsch Sure, checking that some cert file exists should do it.

**Comment by Sergeykot** [comment]
@felixsch yes, I think that should be fine.

**Pull request merged**